.gitignore

```diff
@@ -47,3 +47,4 @@ npm-debug.log
 uploads/
 test_uploads/
 *.mo
+assets/language_options.json
```

Comment:
(in particular, `assets/` is not installed in production, while `static/locale` should be).